plugin-attribution/src/test/java/com/mapbox/common/ShadowLogger.java

```diff
@@ -0,0 +1,33 @@
+package com.mapbox.common;
+
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import org.robolectric.annotation.Implementation;
+import org.robolectric.annotation.Implements;
+
+@Implements(Logger.class)
+public class ShadowLogger {
```

Comment:
We have many ShadowLogger.java distributed in several modules, maybe we could keep it in a test module or something else?

Reply:
I remember trying doing that when logger was first introduced but it appeared to be also not that easy to implement. Perhaps will put some time into it as operational excellence but not now.